pyproject.toml

```diff
@@ -1,19 +1,29 @@
 [tool.black]
-exclude = '''
+force-exclude = '''
 /(
-    \.git
-  | \.hg
-  | \.mypy_cache
-  | \.pytest_cache
+  \.pytest_cache
   | __pycache__
   | _build
-  | buck-out
-  | build
-  | dist
   | migrations
   | static
   | staticfiles
   | templates
-  | venv
 )/
 '''
+line-length = 79
+
+
+# [tool.flake8]
+#   config: .flake8
+# also see: https://gitlab.com/pycqa/flake8/-/issues/428
```

Comment:
Nice style improvement here! I only noticed that sadly this link is broke, probably because the Gitlab repo is just a mirror now and the project is on [Github](https://github.com/PyCQA/flake8), got lost on the way.

For the rest, LGTM.